src/main/java/com/github/dockerjava/jaxrs/ListImagesCmdExec.java

```diff
@@ -29,6 +29,9 @@ public ListImagesCmdExec(WebTarget baseResource) {
 
         if (command.getFilters() != null)
             webResource = webResource.queryParam("filters", urlPathSegmentEscaper().escape(command.getFilters()));
+        
+        if (command.getImageNameFilter() != null)
+        	webResource = webResource.queryParam("filter", urlPathSegmentEscaper().escape(command.getImageNameFilter()));
```

Comment:
put brackets even if contains only one line, because it may end in not obvious errors later